index.js

```diff
@@ -11,14 +11,15 @@ const trimMiddle = (str, length = 16, replaceString = "…") => {
     if (length <= replaceString.length) {
         throw new RangeError("The length argument must be greater than the replaceString length.");
     }
-    if (str.length <= length) { 
-        return str; 
-    } else {
-        let remainder = (length - replaceString.length) / 2;
-        let chunks = [...new Intl.Segmenter().segment(str)].map(x => x.segment);
-        return chunks.slice(0,Math.ceil(remainder)).join('') + 
-        replaceString + 
-        chunks.slice(-Math.floor(remainder)).join('');
+
+    if (str.length <= length) {
+        return str;
     }
+
+    let remainder = (length - replaceString.length) / 2;
+    let chunks = [...new Intl.Segmenter().segment(str)].map(x => x.segment);
+    return chunks.slice(0, Math.ceil(remainder)).join("") +
+            replaceString +
+            chunks.slice(-Math.floor(remainder)).join("");
 };
-export { trimMiddle };
\ No newline at end of file
+export { trimMiddle };
```

Comment:
To avoid duplication errors, we might add a small script to either

a) check that `index.js` and `clientside.js` are largely identical, or

b) generate `index.js` from `clientside.js`

Not sure that's worth the trouble though.

Reply:
I wondered about that, too, or make it a github action. Maybe when a lot of changes come in.